modules/core/include/opencv2/core.hpp

```diff
@@ -2316,11 +2316,11 @@ class CV_EXPORTS PCA
     The operator performs %PCA of the supplied dataset. It is safe to reuse
     the same PCA structure for multiple datasets. That is, if the structure
     has been previously used with another dataset, the existing internal
-    data is reclaimed and the new eigenvalues, @ref eigenvectors , and @ref
+    data is reclaimed and the new @ref eigenvalues, @ref eigenvectors , and @ref
```

Comment:
Comma after word `@ref eigenvectors` can be removed. Same for _read_ and _write_ documentation.